src/client/tile.cpp

```diff
@@ -1175,7 +1175,18 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 					core::rect<s32>(pos_from, dim),
 					video::SColor(255,255,255,255),
 					NULL);*/
-			blit_with_alpha(image, baseimg, pos_from, pos_to, dim);
+
+			v2u32 dim_dst = baseimg->getDimension();
```

Comment:
I would've gone with ``dimension2d<u32>`` since it's what getDimension returns, but I don't really mind either way. What tripped me up is the ``dim != dim_dst``, since I wasn't sure whether ``dimension2d<u32>`` and ``v2u32`` were comparable types. But apparently Irrlicht explicitly defines ``operator==`` and ``operator!=`` between these two types.